Dockerfile

```diff
@@ -193,6 +156,23 @@ USER $USERNAME
 # existing release image build by copying over remaining files from the repo into the container.
 FROM bionemo2-base AS release
 
+WORKDIR /workspace/bionemo2
+
+# Install 3rd-party deps and bionemo submodules.
+COPY ./3rdparty /workspace/bionemo2/3rdparty
```

Comment:
can we set workspace/bionemo2 as some variable for reference in the docker? it used to be BIONEMO_HOME. Or is it automatically set as HOME?

Reply:
So right, actually this is a good point, we should clarify this in the design doc. I would prefer to not copy in `3rdparty` and `subpackages` into the container (likewise with the `docs/` and `scripts/` we copy later).

We could just create a `bionemo` with some `$HOME` directory that users would use? But I think that requires us to 
(1) align on whether it's OK to omit tests and scripts from the image itself, with the expectation that users would mount files into the image they want to run
(2) change CI so that tests are mounted into the container